static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -2182,6 +2192,7 @@ var CStudioForms =
             form.model[repeat.id][0] = [];
 
             this.parentNode.parentNode.reRender(this.parentNode.parentNode);
+            repeatEdited = true;
```

Comment:
What's this for? Don't see it used anywhere in this PR.

Reply:
That was missing in the 'Add first item' case for the repeating group to indicate that there was a change in the form (and for it to show the confirmation dialog when trying to close it after adding a first item in a repeating group.